openvalidation-generation/src/main/resources/java/framework.hbs

```diff
@@ -390,12 +390,16 @@ public List<ValidationRule> Rules = new ArrayList<>();
         return !AT_LEAST_ONE_OF(leftOperand, rightOperand);
     }
 
-    public <T extends Number> Double SUM_OF(T... items)
+    public <T extends Number> BigDecimal SUM_OF(T... items)
```

Comment:
Would it not be far more elegant to use the interface `Number` as return type and ...